### PR TITLE
Make `mcfgv1` and `ctrlcommon` canonical names for importing

### PR DESCRIPTION
**Pull request by LorbusChris**
Entirely code-style.

Not having agreement on the name to use to import the package can lead to not-so-pretty double imports of the same package (as seen here in `container_runtime_config_controller.go`, where the api is imported as both `v1` and  `mcfgv1`).
The same is true for `pkg/controller/common` which is imported with different names across files, and also imported twice in one (`render_controller.go`).

This PR removes the double imports, and canonicalizes the import names as `mcfgv1` and `ctrlcommon` across all files in this repo.

### CODE REVIEW

**Comment by umohnani8** [comment]
LGTM
thanks @LorbusChris!

File: pkg/server/cluster_server.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	clientcmd "k8s.io/client-go/tools/clientcmd"
 	clientcmdv1 "k8s.io/client-go/tools/clientcmd/api/v1"
 
-	v1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
```
Comment (mtrmac) [inline pkg/server/cluster_server.go]: …/pkg/generated/… is a different package; should it use the same prefix?

Comment (LorbusChris) [reply pkg/server/cluster_server.go]: maybe this is the distinction we should make here, call the generated pkg (and only that!) `v1`
Will revert this part.

**Comment by LorbusChris** [comment]
I found another double import of `pkg/controller/common`, which is also imported with different names in different files. Second commit incoming.
/hold

**Comment by LorbusChris** [comment]
@mtrmac the api package will be moved into the api repo eventually -- I believe it is common practice to spell out the names there and not to shorten them.

**Comment by LorbusChris** [comment]
and updated!

/hold cancel

**Comment by runcom** [comment]
loving this - unfortunately we can't merge this right away as we need to wait at least Monday

/hold

**Comment by runcom** [comment]
lifting hold I guess - this has 0 risk

/hold cancel

**Pull request merged**